go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/puppetlabs/pe-sdk-go
+
+go 1.15
```

Comment:
we will have to discuss on which go version we want to work, other go client components are using 1.13 (https://github.com/puppetlabs/pe-client-tools-vanagon/blob/2ce9d08529a9d921c6706e2f4c51bbb5ff30d67a/configs/components/go-binary.rb#L2)